src/zampy/conventions/ALMA.json

```diff
@@ -23,6 +23,16 @@
         "units": "watt/meter**2",
         "positive_dir": "downward"
     },
+    "surface_solar_radiation": {
```

Comment:
I prefer `surface_solar_radiation_downwards`. `incident_shortwave_radiation` is not as explicit as the former one and it also needs surface to indicate this variable is meant for solar radiation at land surface rather than the TOA.